gtda/homology/point_clouds.py

```diff
@@ -10,7 +10,7 @@
 from sklearn.metrics.pairwise import pairwise_distances
 
 from ._utils import _pad_diagram
-from ..externals.python import ripser, SparseRipsComplex
+from ..externals.python import ripser, SparseRipsComplex, CechComplex
```

Comment:
Is this right?

Reply:
Sorry, I don't follow? What is right?

Reply:
I was referring to the import from `externals`. Checking the contents, it is obviously right. I think I am uneasy about mixed naming conventions (Pascal case and lowercase here). But this is not the place to discuss this.